app/src/main/java/ch/epfl/sdp/MapsActivity.java

```diff
@@ -22,23 +22,15 @@ protected void onCreate(Bundle savedInstanceState) {
         Button mapButton = findViewById(R.id.recenter);
         mapButton.setOnClickListener(v -> mapApi.moveCameraOnCurrentLocation());

```

Comment:
For readability and maintainability purposes, I think it should be best to put this code into an initialize method. Here you can see how to code it in several ways and what the advantages are: https://stackoverflow.com/questions/37014156/android-onclicklistener-implementation-best-practices.

Reply:
I read the post and we can do this. Since a lot of those are only temporary buttons, I think we can see that when we put final buttons, but let's note that.